app/src/main/java/com/duckduckgo/app/notification/AndroidNotificationScheduler.kt

```diff
@@ -77,15 +81,29 @@ class NotificationScheduler(
 
         when {
             (!variant().hasFeature(VariantManager.VariantFeature.RemoveDay1AndDay3Notifications) && privacyNotification.canShow()) -> {
-                scheduleNotification(OneTimeWorkRequestBuilder<PrivacyNotificationWorker>(), 1, TimeUnit.DAYS, UNUSED_APP_WORK_REQUEST_TAG)
+                val duration = getDurationForInactiveNotification(PRIVACY_DURATION)
+                scheduleNotification(OneTimeWorkRequestBuilder<PrivacyNotificationWorker>(), duration, TimeUnit.DAYS, UNUSED_APP_WORK_REQUEST_TAG)
             }
             (!variant().hasFeature(VariantManager.VariantFeature.RemoveDay1AndDay3Notifications) && clearDataNotification.canShow()) -> {
-                scheduleNotification(OneTimeWorkRequestBuilder<ClearDataNotificationWorker>(), 3, TimeUnit.DAYS, UNUSED_APP_WORK_REQUEST_TAG)
+                val duration = getDurationForInactiveNotification(CLEAR_DATA_DURATION)
+                scheduleNotification(OneTimeWorkRequestBuilder<ClearDataNotificationWorker>(), duration, TimeUnit.DAYS, UNUSED_APP_WORK_REQUEST_TAG)
             }
             else -> Timber.v("Notifications not enabled for this variant")
         }
     }
 
+    @VisibleForTesting
```

Comment:
I'm ok keeping this for the experiment, but it will be nice if we could refactor this so we don't need `@VisibleForTesting` -> a) test this class in a different way or b) if we want to test this method, we should extract it to a different class.

Reply:
Yeah I was trying to test that by getting the job, but it turns out we can only get the WorkInfo which doesn't include that information at all. 

I think if we go ahead with the experiment all this stuff could be moved into the `canShow()` method of each notification. I also wasn't too happy about the DURATION constants in this class and I believe they should be in each notification class. Will refactor all this if we end up releasing.